ddtrace/internal/_encoding.pyx

```diff
@@ -734,3 +740,167 @@ cdef class MsgpackEncoderV05(MsgpackEncoderBase):
         if ret != 0: return ret
 
         return 0
+
+
+cdef class Packer(object):
```

Comment:
This source file is already quite big. Would it make sense to put this into its own source file?

Reply:
Yeah I considered it but opted not to because, iirc, adding another Cython module significantly increases compilation time 😞. There's also quite a bit of dependence that `Packer` has that is shared with the other encoders (the underlying C struct and all the methods)